forest/main.py

```diff
@@ -247,14 +248,7 @@ def on_change(attr, old, new):
         image_controls.select(name)
         break
 
-    if len(args.files) > 0:
-        navigator = navigate.FileSystem.file_type(
-                args.files,
-                args.file_type)
-    elif args.database is not None:
-        navigator = database
-    else:
-        navigator = navigate.Config(config)
```

Comment:
Am I right in thinking the `config` object encapsulates `args.files` already which is why you could eliminate this block?

Reply:
Yup. :+1: